test/e2e/websocket.spec.ts

```diff
@@ -97,15 +97,14 @@ describe('E2E WebSocket proxy', () => {
 
   describe('with router and pathRewrite', () => {
     beforeEach(() => {
-      // override
-      proxyServer = createApp(
+      const proxyMiddleware = createProxyMiddleware({
```

Comment:
This was a bug with the existing tests: because it didn't reassign `proxyMiddleware`, the `proxyMiddleware.upgrade` access a few lines after this was reading from the wrong object.